.github/workflows/docker-update.yml

```diff
@@ -14,8 +14,26 @@ jobs:
         with:
           username: ${{ secrets.DOCKER_USERNAME }}
           password: ${{ secrets.DOCKER_PASSWORD }}
-      - name: Push to Docker Hub
+      - name: Push to Docker Hub latest
         uses: docker/build-push-action@v2
         with:
           push: true
           tags: profiq/node-java-chrome-ff:latest
+      - name: Get latest chrome version
+        run: |
+          echo ::set-env name=CHROME_VERSION::$(google-chrome --version 2>&1 | head -1 | cut -d'"' -f2 | sed '/^1\./s///' | cut -d'.' -f1 | sed '/Google Chrome /s///')
```

Comment:
set-env is deprecated, you can use following notation to create runtime env variables
https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#setting-an-environment-variable
`echo "NOW=$(date -u +%Y-%m-%dT%H:%M:%S)" >> $GITHUB_ENV` this si now